src/js/rs-release/rs-release.js

```diff
@@ -193,35 +199,29 @@ EvaRsRelease.prototype = {
             }
         });
 
-        var table = '<table id="rs-release-table-by-assembly" class="responsive-table hover tablesorter table-fixed">' +
+        var table = '<table id="rs-release-table-by-assembly-new-data" class="responsive-table hover tablesorter table-fixed">' +
                     '<thead>' +
                     '<tr>' +
-                        '<th>Scientific name</th>' +
-                        '<th>Taxonomy ID</th>' +
+                        '<th>Download link</th>' +
                         '<th>Assembly accession</th>' +
-                        '<th><div title="RS IDs that were clustered in the current release">New Current RS <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that were issued for remapped variants">Remapped clustered <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that were created for new variants">Newly clustered <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that should NOT be used because they are merged into another active RS">Merged RS <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that were split into multiple active RS due to remapping">Split RS <i class="icon icon-generic" data-icon="i"></div></th>' +
+                        '<th><div title="RS IDs that were added in the current release">New Current RS<i class="icon icon-generic" data-icon="i"></div></th>' +
                     '</tr>' +
                     '</thead>' +
                     '<tbody>';
 
-        _.each(releaseData, function (species) {
-            releaseLink = '<a target="_blank" href="' + species.releaseLink + '">' + species.scientificName + '</a>';
-            taxonomyLink = '<a target="_blank" href="' + species.taxonomyLink + '">' + species.taxonomyId + '</a>';
-            assemblyLink = '<a target="_blank" href="' + species.releaseLink + '">' + species.assemblyAccession + '</a>';
-
+        _.each(releaseData, function (assembly) {
+            releaseLink = '<a target="_blank" href="' + assembly.releaseLink + '">FTP link</a>';
+            assemblyLink = '<a target="_blank" href="' + assembly.releaseLink + '">' + assembly.assemblyAccession + '</a>';
```

Comment:
I think we could also remove this release link since we've added the FTP column - similar to how it's been removed for scientific name.
Also a bit inconsistent with how we link the taxonomy ID to ENA, but the assembly accession to our FTP.